pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -94,11 +92,6 @@ func New(imageFsInfoProvider ImageFsInfoProvider, rootPath string, cgroupRoots [
 		cadvisormetrics.OOMMetrics:          struct{}{},
 	}
 
-	// Only add the Accelerator metrics if the feature is inactive
-	if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DisableAcceleratorUsageMetrics) {
```

Comment:
I think this is dropped in https://github.com/kubernetes/kubernetes/pull/114883, right?

Reply:
In fact, this PR does not involve the above changes, right?